Sources/StreamVideo/WebSockets/Client/RetryStrategy.swift

```diff
@@ -43,28 +43,25 @@ struct DefaultRetryStrategy: RetryStrategy {
     @Atomic private(set) var consecutiveFailuresCount = 0
     
     mutating func incrementConsecutiveFailures() {
-        _consecutiveFailuresCount.mutate { $0 += 1 }
+        _consecutiveFailuresCount.mutate { $0 + 1 }
     }
     
     mutating func resetConsecutiveFailures() {
-        _consecutiveFailuresCount.mutate { $0 = 0 }
+        consecutiveFailuresCount = 0
```

Comment:
why do we change this?

Reply:
Because the updated property wrapper doesn't require the mutate function(at least directly). Instead, the setter we are calling here will call `mutate` as expected.